packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -669,13 +682,31 @@ ${sql.join(
     options: {
       asJson?: boolean,
       asJsonAggregate?: boolean,
+      asRaw?: boolean,
       onlyJsonField?: boolean,
       addNullCase?: boolean,
       addNotDistinctFromNullCase?: boolean,
       useAsterisk?: boolean,
     } = {}
   ) {
+    if (this.isSingleFieldSelector) {
+      if (Object.keys(options).length > 0) {
+        throw new Error(
+          "Can't use QueryBuilder.build(options) in isSingleFieldSelector"
```

Comment:
This may not make sense to the person who receives it; maybe something like this?
```suggestion
          "Do not pass options to QueryBuilder.build() when using `buildNamedChildFrom`"
```